vendor/github.com/containerd/continuity/vendor.conf

```diff
@@ -10,4 +10,4 @@ github.com/spf13/pflag 4c012f6dcd9546820e378d0bdda4d8fc772cdfea
 golang.org/x/crypto 9f005a07e0d31d45e6656d241bb5c0f2efd4bc94
 golang.org/x/net a337091b0525af65de94df2eb7e98bd9962dcbe2
 golang.org/x/sync 450f422ab23cf9881c94e2db30cac0eb1b7cf80c
-golang.org/x/sys 665f6529cca930e27b831a0d1dafffbe1c172924
+golang.org/x/sys 77b0e4315053a57ed2962443614bdb28db152054
```

Comment:
✅ Checked; master is already ahead of this commit (this is what's on master https://github.com/golang/sys/compare/77b0e4315053a57ed2962443614bdb28db152054...ac767d655b305d4e9612f5f6e33120b9176c4ad4)